internal/util/debug/debug.go

```diff
@@ -74,9 +76,15 @@ func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *za
 
 	must.NoError(r.Register(requestCount))
 
+	// started handler, which is used for startup probe, returns StatusOK when FerretDB listener were initialized.
```

Comment:
That line describes some "listener" the debug handler knows nothing about. All it knows is that the "started handler" should return different values depending on whether the "started" channel is closed or not.